desktop/release-notes.md

```diff
@@ -24,12 +24,118 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+>
+> {% include desktop-install.html %}
+
+## New
+
+- Added the experimental docker daemon with the containerd image store.
+- Allow Docker Desktop to run untagged images. Fixes [https://github.com/docker/for-mac/issues/6425]
+- Add search capabilities to Docker Extension Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).
+- Copied terminal contents do not contain non-breaking spaces anymore.
+- Enabled Compose V2 by default on new installations of Docker Desktop.
+- Added View menu with Zoom In, Zoom Out, and Actual Size options to control the size of the Docker Desktop UI.
+- Added compose stop button if any related container is stoppable.
+- Individual compose containers are now deletable from overview.
+- Remove workaround for virtiofsd <-> qemu protocol mismatch on Fedora 35, as it is no longer needed. Fedora 35 users should upgrade the qemu package to the most recent version (qemu-6.1.0-15.fc35 as of the time of writing).
+- Implemented an integrated terminal for containers.
+- Show address for external links.
+
+### Upgrades
+
+- [Docker Compose v2.10.2](https://github.com/docker/compose/releases/tag/v2.10.2)
+- [Docker Compose "Cloud Integrations" v1.0.29](https://github.com/docker/compose-cli/releases/tag/v1.0.29)
+- [Docker Scan v0.19.0](https://github.com/docker/scan-cli-plugin/releases/tag/v0.19.0)
+- [Kubernetes v1.25.0](https://github.com/kubernetes/kubernetes/releases/tag/v1.25.0)
+- [Go 1.19](https://github.com/golang/go/releases/tag/go1.19)
+- [cri-dockerd v0.2.5](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.2.5)
+- [Buildx v0.9.1](https://github.com/docker/buildx/releases/tag/v0.9.1)
+- [containerd v1.6.8](https://github.com/containerd/containerd/releases/tag/v1.6.8)
+- [containerd v1.6.7](https://github.com/containerd/containerd/releases/tag/v1.6.7)
+- [runc v1.1.4](https://github.com/opencontainers/runc/releases/tag/v1.1.4)
+- [runc v1.1.3](https://github.com/opencontainers/runc/releases/tag/v1.1.3)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Enable Compose V2 after factory reset.
+- Upgraded kernel to 5.10.124
+- Improved overall performance issues caused by calculating disk size. Related to [docker/for-win#9401](https://github.com/docker/for-win/issues/9401).
+- Prevent users on ARM macs without Rosetta installed from switching back to Compose V1, which has only intel binaries.
+- Descending default sort order for volumes size and created column, containers started column.
+- Re-organized container row actions.
+- Run every command on the Quickstart guide immediately.
+- Defined sort order for container/compose state column running > some running > paused > some paused > exited > some exited > created.
+- Fixed issues with the image list appearing empty in Docker Desktop even though there are images. Related to [docker/for-win#12693](https://github.com/docker/for-win/issues/12693) and [docker/for-mac#6347](https://github.com/docker/for-mac/issues/6347).
```

Comment:
with this bug fix, i didn't note another change in behavior that could be noticeable to users. would it be possible to add it?
```
Defined what images are "in use" based on whether or not system containers are displayed; if system containers related to Kubernetes and Extensions are not displayed, the related images will not be defined as "in use."
```
(it's a difficult change to articulate succinctly, so feel free to modify if my phrasing is confusing!)

Reply:
Small tweak to your suggestion:
Defined what images are "in use" based on whether or not system containers are displayed. If system containers related to Kubernetes and Extensions are not displayed, the related images are not defined as "in use."

Reply:
Added your revised description @aevesdocker

Reply:
thank you both!